Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityInputActionRulesInspector.cs

```diff
@@ -97,11 +97,6 @@ public override void OnInspectorGUI()
                 !MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled ||
                  MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.InputActionsProfile == null)
             {
-                if (GUILayout.Button("Back to Configuration Profile"))
```

Comment:
This is the correct behavior.

Reply:
Agreed, this is how it is used on al the profiles.